src/test/java/org/mule/extension/sftp/SftpServer.java

```diff
@@ -85,6 +85,10 @@ public void start() {
     }
   }
 
+  public Boolean isStarted() {
+    return sshdServer != null && sshdServer.isStarted();
+  }
+
```

Comment:
I remember this discussion, are you using this method? Where?

Reply:
Ohh.. no, I'm not, I'll remove it

Reply:
Done